tests/unit/test_io_utils.py

```diff
@@ -173,6 +175,61 @@ def test_copy_files_to_directory(self, tmp_path):
         print(copied_file)
         assert copied_file.exists(), "Temp folder should have been created"
 
+        # 'tmp_path' is a default pytest fixture
+
+    @pytest.mark.parametrize(
+        "workflow_id, column_to_update, update_value, should_fail",
+        [
+            ["b3b197b3-fdca-4647-9fd8-bf16d2cb734d", "WONK", "wonderwoman", True],
+            ["b3b197b3-fdca-4647-9fd8-bf16d2cb734d", "ALIAS", "wonderwoman", False],
+            ["b3b197b3-fdca-4647-9fd8-bf16d2cb734d", "STATUS", "Failed", False],
+            ["682f3e72-0285-40ec-8128-1feb877706ce", "WDL_NAME", "Calm.wdl", True],
+        ],
+    )
+    def test_set_alias_for_workflow_id(
```

Comment:
this test name is little out of date: I think you are testing the general behavior of attempting to change the TSV, not just changing the alias.
So, I'd say update the name of the function, and set up more test cases.

Reply:
done